README.rst

```diff
@@ -117,116 +210,10 @@ For example, the example above can read information about the
      args = parser.parse_args()
      ...
 
-List of provided environment variables by SageMaker Containers
```

Comment:
I think this header is linked to by other READMEs, so it'd be good to update those after this change